pyproject.toml

```diff
@@ -60,3 +60,14 @@ extend-select = [
 docstring-quotes = "double"
 inline-quotes = "single"
 multiline-quotes = "single"
+
+[tool.uv]
+dev-dependencies = [
```

Comment:
They recommend not to use this field anymore: https://docs.astral.sh/uv/reference/settings/#dev-dependencies
Instead it is proposed to use `dependency-groups.dev`: https://peps.python.org/pep-0735/